src/formatDate.js

```diff
@@ -8,7 +8,48 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  let dateArr = [];
+  let resultString = '';
+  let index = 0;
+
+  if (date.includes('-')) {
+    dateArr = date.split('-');
+  } else if (date.includes('/')) {
+    dateArr = date.split('/');
+  } else {
+    dateArr = date.split('.');
+  }
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
+    const fromIndex = fromFormat.indexOf(toFormat[i]);
+
+    index++;
+
+    if (toFormat[i] === 'YY' && fromFormat.includes('YYYY')) {
+      const number = dateArr[index - 1];
+
+      resultString += number.slice(2);
+    } else if (toFormat[i] === 'YYYY' && fromFormat.includes('YY')) {
+      const year = Number(dateArr[index - 1]);
+
+      if (dateArr[index - 1] === '00') {
+        resultString += '2000.';
+        continue;
+      } else {
+        const fullYear = year < 30 ? `20${year}` : `19${year}`;
+
+        resultString += fullYear;
+      }
```

Comment:
When you're converting from 'YY' to 'YYYY', your code doesn't handle the case where the year is less than 10 correctly. For example, if the year is '09', your code will convert it to '2009', which is incorrect. It should be '1909'. Consider adding a check for this case.